devtools/prod-envs/qcfractal.yaml

```diff
@@ -0,0 +1,11 @@
+name: qcfractal
+channels:
+  - defaults
+  - conda-forge
+dependencies:
+  - python
+  - qcfractal
```

Comment:
Should this one also contain `qcfractal-dashboard`?

Reply:
I would say not until this is stable, but then yea lets include it. Can we cap `python = 3.7`?